### PR TITLE
Feature/#28 add map and marker components

### PR DESCRIPTION
**Pull request by elpollo12**
### Description

Added map markers for dasher and user.

fixes # (issue)

#28 

#### Screenshots

scrnli_30_10_2021 13-39-22

### CODE REVIEW

File: src/layouts/Map.styled.jsx [hunk 1]
```diff
@@ -51,16 +53,15 @@ const BackTxt = styled.p`
...
   color: #f7cd46;
-  font-size: 1em;
+  font-size: 1.7em;
   font-weight: bold;
   display: flex;
   flex-direction: column;
   margin: auto 0;
-
-  @media (min-width: 960px) {
+  height: auto @media (min-width: 960px) {
```
Comment (Israel-Laguan) [inline src/layouts/Map.styled.jsx]: You forgot a `;`, please check this rule

File: src/layouts/Map.styled.jsx [hunk 2]
```diff
@@ -156,7 +157,7 @@ const FooterMapAddress = styled.div`
 
   @media (min-width: 960px) {
     font-size: 1em;
-    padding: 3.78% 0;
+    padding: 2% 0;
```
Comment (Israel-Laguan) [inline src/layouts/Map.styled.jsx]: Please avoid percentage like a plague. Please don't introduce new relative padding/margin/height, even width should avoid relative sizes when possible. If you introduce relative sizes please be sure to add stop values (`min-`/`max-`).

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]
Please fix all those relative/percentage sizing

**Comment by elpollo12** [comment]
Updated code

**Review by Israel-Laguan — APPROVED** [review]
Well, vh/vw are basically percentages too, but we can merge.

Please try to investigate more on the use of relative units.

**Pull request merged**